som

```diff
@@ -105,7 +105,8 @@ parser.add_argument('-T', '--no-trace', help='do not print truffle compilation i
 parser.add_argument('--no-graph-pe', help='disable Graph PE',
                     dest='graph_pe', action='store_false', default=True)
 
-
+parser.add_argument('-O', '--no-ansi', help='Turns off ANSI coloring for output from interperter',
+                    dest='no_ansi_coloring', action='store_true', default=False)
```

Comment:
Please avoid negation where possible.
So, ` dest='ansi_coloring', action='store_false', default=True`. This is much more explicit and easier to read.